carthage.mk

```diff
@@ -0,0 +1,45 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+ifneq (,$(findstring darwin,$(MEMO_TARGET)))
+
+SUBPROJECTS      += carthage
+CARTHAGE_VERSION   := 0.37.0
+DEB_CARTHAGE_V     ?= $(CARTHAGE_VERSION)
+
+carthage-setup: setup
+	wget -q -O $(BUILD_SOURCE)/Carthage-$(CARTHAGE_VERSION).tar.gz https://github.com/Carthage/Carthage/archive/$(CARTHAGE_VERSION).tar.gz
```

Comment:
Please remove this line, the download is handled by `GITHUB_ARCHIVE`

Reply:
My bad, I thought I had deleted that line. Just pushed a commit that fixes this